gradle/java.gradle

```
@@ -278,7 +278,7 @@ def isJdkForced(String javaName) {
 }
 
 def isTestingInstrumentation(Project project) {
-  return project.name == "junit-4.10"
+  return project.name == "junit-4.10" || project.name == "testng-6.4"
```

```suggestion
  return [ "junit-4.10", "testng-6.4" ].contains(project.name)
```

(or something like this... I assume there will be more testing projects.)